ci/parse_ci_envs.sh

```diff
@@ -8,11 +8,11 @@ set -uex
 if [ -n "${STAGE_NAME:?}" ]; then
   case $STAGE_NAME in
     *CentOS\ 7*|*el7*|*centos7*)
-      : "${CHROOT_NAME:=epel-7-x86_64}"
+      : "${CHROOT_NAME:=centos+epel-7-x86_64}"
       : "${TARGET:=centos7}"
       ;;
-    *CentOS\ 8*|*el8*|*centos8*)
-      : "${CHROOT_NAME:=epel-8-x86_64}"
+    *CentOS\ 8*|*EL 8*|*el8*|*centos8*)
```

Comment:
(lint) Couldn't parse this case item. Fix to allow more checks. [SC1073]
(lint) Expected ) to open a new case item. Fix any mentioned problems and try again. [SC1072]
(lint) Did you forget to move the ;; after extending this case item? [SC1085]